### PR TITLE
corrected grammar and add to QA_testing.md

### PR DESCRIPTION
**Pull request by Maxlyu254**
I have made some corrections and added some information to the QA_testing.md. The added information is mostly about what is the job of a QA engineer like and how to write decent QA tests with undergraduate-level knowledge. Please tell me if I need to add something more to the topic to help the readers understand the topic better.

### CODE REVIEW

**Review by yi1z — APPROVED** [review]
Overall great work. Changes proposed in the comments are optional, and are all minor changes which do not impact much.

**Comment by Maxlyu254** [comment]
I have reduced the lists that are too long and explained that I included the summary of the articles.

File: Topics/Development_Process/QA_testing.md [hunk 1]
```diff
@@ -1,43 +1,64 @@
...
+- Performance Testing Engineer: Performance testing engineers need to evaluate the efficiency of software in different conditions.
+- Web QA Engineer: Web QA engineers focus on testing web applications and websites. They make sure that the application works well across different web browsers (cross-browser compatibility), different devices (cross-device compatibility), and different window sizes (responsiveness).
+- Accessibility Testing Engineer: Accessibility testing engineers ensure that software applications can be accessed by people with different kinds of disabilities.
+- Continuous Integration/Continuous Deployment QA Engineer (CI/CD QA Engineers): CI/CD QA engineers focus on integrating quality assurance processes into automated build and deployment pipelines so that the tests can be conducted rapidly.
+
+As we can see from above, QA engineers are more than simply writing automated test cases and doing manual tests one after another. Sometimes they require more knowledge than simply knowing how to code, like the accessibility testing engineers. They are also an important part of delivering a reliable and high-quality product.
```
Comment (yin0409) [inline Topics/Development_Process/QA_testing.md:51]: nice lis for this problem

File: Topics/Development_Process/QA_testing.md [hunk 2]
```diff
@@ -1,43 +1,64 @@
...
```
Comment (yin0409) [inline Topics/Development_Process/QA_testing.md:50]: the list create pretty well, good understanding of each.

File: Topics/Development_Process/QA_testing.md [hunk 3]
```diff
@@ -46,11 +67,47 @@ There are many reasons why software testing is important. This blog [7 reasons w
...
+- Who should write the QA test cases: It's best to have people not involved in coding the product write the test cases because it might bring in fresh perspectives.
+
+If you need more help on how to write a test case, here is a [short blog](https://blog.testlodge.com/how-to-write-test-cases-for-software-with-sample/) that's relatively easy to understand. This blog provides visual examples of what a test case should look like, and a [short video](https://www.youtube.com/watch?v=khGa1Rdzd2A) showing step-by-step how to write a test case.
 
 ## Automated testing
 
-[This article](./Automated_Testing.md) provides a nice overview of automated testing, which is an important tool in QA testing.
+Automated testing is an important part of QA testing, that involves engineers writing test scripts, and running them to get test results. Since the test scripts can be reused, it's extremely suitable for conducting tests repeatedly over many iterations of a product. Sometimes, there is a dedicated QA team that writes automated tests. Sometimes, the software development team will do the work.
+
```
Comment (yin0409) [inline Topics/Development_Process/QA_testing.md:112]: it is better to make a list to illustrate why the automated testing is important. and readable?

**Review by yin0409 — COMMENTED** [review]
Nice and readable md file.  there still some problem to enhance the feeling to the reader.

**Pull request merged**